### ISSUE TITLE
NPE on particles

### ISSUE THREAD

**Issue by Drullkus** [Samuel-Harbord/SpectriteMod#26]
No idea how to reproduce issue. I'm crashing when I try to join the BTM Server.

`spectrite-1.6.3a.jar`

Running forge `14.23.0.2544`

[crash-2017-11-17_12.08.17-client.txt](https://github.com/Samuel-Harbord/Spectrite/files/1483664/crash-2017-11-17_12.08.17-client.txt)

**Comment by Flashfyre** [comment]
I got at least one other report of this, yeah. Not sure why it's happening, I've never had this issue. Not sure if it's too late to make a patch for this for BTM but I'd probably need to reproduce it to know if it'd work.

**Comment by SihenZhang** [comment]
I got a similar but **different** crash report when I rejoined the BTM server after I disconnected from the server.

Here is the crash report: https://paste.ee/p/rmG8d

**Comment by Flashfyre** [comment]
Yeah it's basically the same thing, I already fixed (worked around) this but an update hasn't been made yet. I couldn't replicate this in the end so I missed it the first time.
